packages/vitest-angular/package.json

```
@@ -29,7 +29,7 @@
   },
   "peerDependencies": {
     "@analogjs/vite-plugin-angular": "^1.3.0",
-    "@angular-devkit/architect": "^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0",
+    "@angular-devkit/architect": "0.1500.0-0.1599.999 || 0.1600.0-0.1699.999 || 0.1700.0-0.1799.999 || 0.1800.0-0.1899.999",
```

Why not use the caret ranges?

```suggestion
    "@angular-devkit/architect": "^0.1500.0 || ^0.1600.0 || ^0.1700.0 || ^0.1800.0",
```